pages/people/[id]/index.tsx

```diff
@@ -34,7 +34,12 @@ const PersonPage = (): React.ReactElement => {
             {canViewArea(user) ? <WarningNotes id={personId} /> : <></>}
             <PersonDetails person={person} />
             <AllocatedWorkers person={person} />
-            {canViewArea(user) ? <Relationships id={personId} /> : <></>}
+            {user.hasDevPermissions ||
```

Comment:
I've been modelling these permission checks as functions in this file – https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/main/lib/permissions.ts

So in this case it'd be something like (⚠️ pseudo code!):

```ts
const canUserSeeRelationshipsForPerson = (user: User, person: Resident) => {
  if(user.hasDevPermissions) {
    return true;
  }

  if(user.hasAdultPermissions && person.contextFlag == 'A') {
    return true;
  }

  return false;
}
```

A few main things here:
1) This is a pure function, only operating on inputs, which is nice in terms of re-use
2) We can re-use these functions more easily
3) More explicit condition checks make it easier to read later to understand why access was or wasn't granted
4) You can unit test the permission check explicitly as a separate func.
5) We will eventually be moving some of these permissions (either to the backend, or likely to some more common front end place), as part of the permissioning work, so putting all the checks nicely in one place should make it a little easier to migrate 😄

Reply:
Hi @chrishutchinson,

sounds good, will definitely use this structure for the permissions!
I'll move also the pre-existing ones in this file (the "canViewArea" function)

Reply:
Nice 👍 and yeah sounds good, `canViewArea` isn't well named either, so a good opportunity to tidy that up 😄

Reply:
I've renamed `canViewArea` to `devCanViewArea` :)